src/main/java/com/extendedclip/papi/bungeeexpansion/BungeeExpansion.java

```diff
@@ -73,6 +73,11 @@ public Map<String, Object> getDefaults() {
     public String onRequest(final OfflinePlayer player, String identifier) {
         final int value;
 
+        if (identifier.startsWith("online_")) {
+            final String server = identifier.substring(7).toLowerCase();
+            return counts.containsKey(server) ? "§aOnline" : "§cOffline";
```

Comment:
I feel like this should be some config strings, to allow configuration from the user.
And by default should it be without color in case a user wants to use it to check the state.

Reply:
Hey Andre, Thank you for the suggestion!

I’ve made the changes as per your recommendations:
- Added configurable strings in the config file for the online/offline messages.
- By default, the messages will be without color, allowing users to customize them according to their needs if they want to display the status in a different format.